template/python3-http-debian/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM openfaas/of-watchdog:0.7.2 as watchdog
+FROM python:3.7-slim-buster
+
+COPY --from=watchdog /fwatchdog /usr/bin/fwatchdog
+RUN chmod +x /usr/bin/fwatchdog
+
+ARG ADDITIONAL_PACKAGE
+# Alternatively use ADD https:// (which will not be cached by Docker builder)
```

Comment:
```suggestion
```

Reply:
Needs to be removed. I made this comment earlier